### PR TITLE
Move Windows binary to bintray

### PR DESCRIPTION
**Pull request by tkelman**
Since sourceforge is horribly unreliable.

also fixup REQUIRE

### CODE REVIEW

**Comment by acroy** [comment]
Thanks! I think the changes are uncontroversial. But do you know if the failure on AppVeyor related to 0.4 or windows?

**Comment by tkelman** [comment]
Probably 0.4, similarly to what's happening on Travis. We could adjust the AppVeyor setup to build only on stable, or both stable and nightly.

**Comment by ViralBShah** [comment]
What about just putting them on S3?

**Comment by tkelman** [comment]
I only have indirect access to S3. The prefix url ends up caching these to S3 since they're added to a whitelist at https://github.com/staticfloat/cache.julialang.org

**Comment by acroy** [comment]
@tkelman It would be good to have AppVeyor test for stable. I don't have access to a Windows machine to test, although I doubt that your changes cause any problems...

**Comment by tkelman** [comment]
Okay, I adjusted appveyor.yml to the latest recommended version which will test on both nightly and 0.3. I'm not sure why, but it looks like this repository was enabled in appveyor by both @tshort and @scidom, and the status links are occasionally overwriting one another. I can probably disable one of the webhooks, if you don't mind. Also it would be good to contact appveyor's support address and ask for your accounts to be upgraded to the newer free-and-fast plan, since these builds are looking pretty slow and taking a while to queue.

**Comment by acroy** [comment]
Thanks again. I guess I can merge this now? It would be great if you could look into the webhooks. Who should ask the support for the upgrade?

**Comment by tkelman** [comment]
I don't think this makes the situation any worse, Travis and Appveyor are all verifying that things are still working on 0.3 and still broken on 0.4. I disabled the webhook that I think was going to @tshort's AppVeyor account since I'm pretty sure it's redundant. If @scidom could send an email to team@appveyor.com asking to have his account moved to the faster VM's (ref http://www.appveyor.com/blog/2015/06/23/new-oss-build-environment-and-xamarin-support) then future builds should run faster and spend less time queued.

**Comment by tshort** [comment]
Thanks @tkelman for figuring that out...

File: deps/build.jl [hunk 1]
```diff
@@ -18,7 +18,7 @@ sundialslibs = enable_sensitivities ? [cvodes, idas, kinsol, nvecserial] : [cvod
 sundialsver = "sundials-2.5.0"
 provides(Sources, URI("http://ftp.mcs.anl.gov/pub/petsc/externalpackages/$sundialsver.tar.gz"), sundialslibs)
 
-provides(Binaries, URI("http://sourceforge.net/projects/juliadeps-win/files/$sundialsver.7z"),
+provides(Binaries, URI("https://cache.e.ip.saba.us/https://bintray.com/artifact/download/tkelman/generic/$sundialsver.7z"),
```
Comment (stevengj) [inline deps/build.jl]: Why is the cache needed on top of bintray?

Comment (tkelman) [inline deps/build.jl]: It might not be, I just haven't decided yet which is going to be more reliable overall, the cache site or bintray by itself.

**Pull request merged**